templates/patroni.yml.j2

```diff
@@ -19,7 +19,7 @@ log:
   file_size: 600
 
 restapi:
-  listen: '{{ self_ip }}:8008'
+  listen: '0.0.0.0:8008'
```

Comment:
I think it will produce issues with [juju spaces](https://juju.is/docs/juju/juju-spaces) once we are ready.
Anyway LGTM for now. 

@delgod any security concerns here?